plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/messages/ModelMessages.java

```diff
@@ -125,6 +125,7 @@ public class ModelMessages extends NLS {
 	public static String model_navigator_Model_root;
 	public static String model_navigator_Name;
 	public static String model_navigator_Project;
+	public static String model_navigator_folder;

```

Comment:
Please, fix tabs and incorrect indentation level (should be 4) for your string in this file.